source/production/F0.Minesweeper.Logic/Utilities.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic
+{
+	static class Utilities
+	{
+		internal static IEnumerable<Location> GetLocationsAreaAroundLocation(IEnumerable<Location> allLocations, Location location, bool exludeGivenLocation)
+		{
+			var returnLocations = new List<Location>();
```

Comment:
We try to get the location area around a given location so we could maybe rename this to e.g. `locationArea` so that we always know what this represents

Reply:
Yep, thats a better name. Thanks 😄